package.json

```diff
@@ -32,6 +32,7 @@
     "@actions/core": "^1.10.0",
     "@actions/glob": "^0.2.0",
     "@actions/io": "^1.1.1",
+    "@babel/core": "^7.23.2",
```

Comment:
btw, why is that module being required now?

Reply:
This change is related to the [vulnerability alert](https://github.com/vmware-labs/vmware-image-builder-action/security/dependabot/15)